src/main/java/org/spongepowered/common/mixin/core/network/rcon/MixinRConThreadClient.java

```diff
@@ -87,7 +88,7 @@ public InetSocketAddress getVirtualHost() {
     @Inject(method = "closeSocket", at = @At("HEAD"))
     public void rconLogoutCallback(CallbackInfo ci) {
         if (this.loggedIn) {
-            SpongeImpl.getScheduler().callSync(() -> {
+            ((SpongeScheduler) Sponge.getServer().getScheduler()).callSync(() -> {
```

Comment:
Make this a method already..... skip the casting....

Reply:
This was before some refactoring.